libp2p/protocols/identify.nim

```diff
@@ -107,29 +121,44 @@ proc decodeMsg*(buf: seq[byte]): Option[IdentifyInfo] =
       iinfo.protoVersion = some(protoVersion)
     if r6.get():
       iinfo.agentVersion = some(agentVersion)
+    if r8.get() and r1.get():
+      if iinfo.pubkey.get() == signedPeerRecord.envelope.publicKey:
+        iinfo.signedPeerRecord = some(signedPeerRecord.envelope)
     debug "decodeMsg: decoded identify", pubkey = ($pubkey).shortLog,
```

Comment:
This `debug` print looks more complex than everything else in this block, maybe we should factor it out to a template or something, because it definitely threw me off at first.

Reply:
Fixed